src/content/platform-integration/ios/ios-latest.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Flutter on iOS
+description: Learn about Flutter on the latest release of iOS
+---
+
+You can develop Flutter on the iOS platform, even on
+the latest release of iOS. The latest Flutter SDK
+already supports a number of the features in the
+latest release of iOS.
+
+Of course, if you find a bug on Flutter,
+please [file an issue][].
+
+[file an issue]: {{site.github}}/flutter/flutter/issues
+
+## iOS 18 release
+
+| Feature                                        | Status                                                                                                        |  
+|------------------------------------------------|---------------------------------------------------------------------------------------------------------------|
+| Apple intelligence (AI)                        | Not yet tested. On Flutter roadmap.                                                                           |
+| Control Center (CC) widgets                    | Tested. Info added to [extension page][]. Requires writing some Swift code to create a toggle for the CC.     |
+| Light/dark/tinted icon colors                  | Tested. Info added to [deployment page[].                                                                     | 
+| Locked and hidden apps                         | Tested. Just works.                                                                                           |
+| iOS formatting menu                            | On Flutter roadmap. Not yet available.                                                                        |
+|**Translate** support for the context edit menu | On Flutter roadmap. Not yet available.                                                                        |
+| Target and build for iOS 18                    | Tested. Should just work.                                                                                     |
+| Cupertino widget A and B                       | New!                                                                                                          |
```

Comment:
This is still placeholder text.

 I don't think this chart is quite ready, for example "should just work" doesn't sound very confident, I don't particularly want to frame it like this.

I definitely want to land removing the "beta" references on the other two pages, but can we hold off landing this new page until we've refined the wording?